docs/generated/sql/bnf/stmt_block.bnf

```diff
@@ -1149,6 +1149,7 @@ unreserved_keyword ::=
 	| 'ESCAPE'
 	| 'EXCLUDE'
 	| 'EXCLUDING'
+	| 'EXCLUDED_USER_COLUMNS'
```

Comment:
instead of adding a new keyword, could we use existing keywords? (e.g. the option could be `EXCLUDE COLUMNS`). Do you agree @dt ?

Reply:
No preference on my side. Guessing we avoid adding keywords when we can?

Reply:
yeah EXCLUDE (and I guess COLUMNS) is better than adding a new keyword. the underscores are also not very sql-y.

Reply:
Fixed